### PR TITLE
add yell and linkshell2 support

### PR DESCRIPTION
**Pull request by arshesney**
Added filtering for yell and linkshell2 messages, the raw yells have some unicode characters wrapping sender name that require a different matching pattern

### CODE REVIEW

File: addons/blist/blist.lua [hunk 1]
```diff
@@ -254,7 +256,8 @@ end)
 windower.register_event('incoming text',function (original, modified, mode)
 	if settings.useblist == true then
 		name = "blist"
-		if mode == 14 and settings.linkshell == true then -- linkshell (others)
+		--print(tostring(mode))
```
Comment (z16) [inline addons/blist/blist.lua]: Can remove the `print`.

**Review by z16 — COMMENTED** [review]
Looks good, just remove the one comment. Also, do you think it makes sense to split up LS1 and LS2? I'm ambivalent on the issue.

**Comment by arshesney** [comment]
I though about splitting the two linkshells, but, at least from my point of view, if I blacklist someone I wouldn't want him popping in other spots. Yet you might be right and having both as a separate option is more in line with the original idea.

**Comment by z16** [comment]
This is good for now, I wasn't even advocating for splitting them up, I was just curious. I'll leave that decision up to you/the community.

**Pull request merged**